component/authentication-endpoint/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/tiqr/QRCode.java

```diff
@@ -203,7 +203,9 @@ private String sendRESTCall(String url, String urlParameters, String formParamet
             }
             return TiqrConstants.FAILED + e.getMessage();
         } finally {
-            connection.disconnect();
+            if (connection != null) {
```

Comment:
HttpURLConnection is not extends from AutoClosable interface, so we can't use it with try with resource.